library/utils.c

```diff
@@ -201,3 +204,16 @@ void hexDump (char *desc, void *addr, int len) {
     fflush(stdout); 
     ZITI_LOG(DEBUG, " ");
 }
+
+void ziti_log_format(char* time_str, size_t time_str_len, uv_timeval64_t start_time) {
+#if _WIN32
+    time_t rawtime = start_time.tv_sec;
+    struct tm  start_tm;
+
+    start_tm = *localtime(&rawtime);
+    strftime(time_str, time_str_len, "%Y-%m-%dT%H:%M:%S", &start_tm);
```

Comment:
@rentallect - i pust `strftime` back but with an updated formatter. the `%FT%T` format crashed on that line for me as well but with the different format string it doesn't crash. looking over at https://docs.microsoft.com/en-us/cpp/c-runtime-library/reference/strftime-wcsftime-strftime-l-wcsftime-l?view=vs-2019 - I will try using `%F%T` which seems to be the equivalent of `%FT%T` in microsoft land

Reply:
after refactoring a bit - i was not able to reproduce any sort of crash using the original code. i refactored after I noticed i'd used localtime instead of gmtime by mistake and then the code was identical for win32 and non. 

I did read in the ms doc this snippet ```Both the 32-bit and 64-bit versions of gmtime, mktime, mkgmtime, and localtime all use one common tm structure per thread for the conversion``` could you have maybe had called this function on different threads causing a crash? This code is only called one time during log initialization so i'm inclined to leave it the same for both _WIN32 and not